Assets/Scripts/Game/Items/ItemCollection.cs

```diff
@@ -96,8 +96,9 @@ public float GetWeight()
             float weight = 0;
             foreach (DaggerfallUnityItem item in items.Values)
             {
-                // Horses, carts and arrows are not counted against encumbrance.
-                if (item.ItemGroup != ItemGroups.Transportation && item.TemplateIndex != (int)Weapons.Arrow)
+                // Horses and carts are not counted against encumbrance.
+                // item.TemplateIndex != (int)Weapons.Arrow must have been there for lack of stack splitting
```

Comment:
No arrows weight is not counted against encumbrance in classic. I tested this quite thoroughly I think. I will check again when I get a chance.